src/main/java/network/kngsgaming/minecraft/EnchantEvents.java

```diff
@@ -19,14 +19,14 @@
 import java.util.Map;
 
 public class EnchantEvents implements Listener {
-    private VanillaEnchants plugin;
+//     private VanillaEnchants plugin;
```

Comment:
You can change sendToConsole() to static and use it like
`VanillaEnchants.sendToConsole(msg);`
I see no problems with privacy since it's a general bukkit api method. Or am I wrong?

Reply:
Oh! yes you are right.